changelogs/fragments/9604-mtu-allow-ipipgre-interfaces.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - nmcli - allow setting ``MTU`` parameter for ``ipip`` and ``gre`` interface types (https://github.com/ansible-collections/community.general/pull/9604).
```

```suggestion
  - nmcli - allow setting ``mtu`` parameter for ``ipip`` and ``gre`` interface types (https://github.com/ansible-collections/community.general/pull/9604).
```
It might also be useful to adjust the `mtu` option's documentation. Right now it says "Can be used when modifying Team, VLAN, Ethernet (Future plans to implement wifi, gsm, pppoe, infiniband).".